packages/webpack/css-extract-webpack-plugin/test/hotCases/css/basic/rspack.config.js

```diff
@@ -11,7 +11,7 @@ import path from 'node:path'
 /** @type {import('webpack').Configuration} */
 export default {
   output: {
-    publicPath: './',
+    publicPath: 'http://localhost:3000/',
```

Comment:
Why do we need this?

Reply:
It seems the test would also pass if reverted to `'./'`
<img width="692" height="205" alt="image" src="https://github.com/user-attachments/assets/2da8c4e0-7191-481d-878a-10749b55f5c7" />